scripts/build-manifest/create.js

```diff
@@ -3,15 +3,14 @@
 const path = require('path');
 const sortChunks = require('webpack-sort-chunks').default;
 const escapeRegExp = require('lodash/escapeRegExp');
-const castArray = require('lodash/castArray');
 const merge = require('deepmerge');
 
-function parseWebpackStats(stats) {
-    const { hash, publicPath } = stats;
+function parseWebpackStats(statsJson) {
+    const { hash, publicPath } = statsJson;
```

Comment:
as you're extracting `hash` and `publicPath` here, maybe also extract `chunks` and use it on line 13?

Reply:
Done!